### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by snafuz**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/---feature-request.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+### Description
+
+<!--- Please leave a helpful description of the feature request here. --->
+
+### New or Affected Resource(s)
+
+<!--- Please list any new or affected resources and data sources that are part of the request. For example: "oci_core_vcn" --->
+
+
+### Potential Terraform Configuration
```
Comment (Djelibeybi) [inline .github/ISSUE_TEMPLATE/---feature-request.md:29]:
```suggestion
### Example Terraform configuration
```

File: .github/ISSUE_TEMPLATE/---feature-request.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+<!--- Information about code formatting: https://help.github.com/articles/basic-writing-and-formatting-syntax/#quoting-code --->
+
+```hcl
+# Copy-paste any Terraform configurations for how the requested feature may be used.
```
Comment (Djelibeybi) [inline .github/ISSUE_TEMPLATE/---feature-request.md:34]:
```suggestion
# Provide example Terraform configurations for how the requested feature may be used.
```

**Review by Djelibeybi — COMMENTED** [review]
I'm not going to flag this as request changes, as these are all suggestions.

**Review by hyder — APPROVED** [review]
Approved

**Review by hyder — APPROVED** [review]
Approved

**Pull request merged**